src/gesv_mixed_gmres.cc

```diff
@@ -266,19 +276,24 @@ int64_t gesv_mixed_gmres(
 
                 // Wj1 = M^-1 A Vj
                 slate::copy( Vj, X_lo, opts );
+                Timer t_getrs_lo_2;
                 getrs( A_lo, pivots, X_lo, opts );
+                timers[ "gesv_mixed_gmres::getrs_lo" ] += t_getrs_lo_2.stop();
                 slate::copy( X_lo, Wj1, opts );
 
+                Timer t_gemm_lo_2;
```

Comment:
You could reuse the `t_gemm_lo_1` (which should be renamed `t_gemm_hi` or `t_gemm_hi_1`), by doing `t_gemm_hi.start();` instead of a new `t_gemm_lo_2` (renamed `t_gemm_hi_2`). Same in other places. Your choice.

Reply:
I went with the `.start()` option. I think it is a cleaner implementation compared to using `_#`.